src/rust/engine/fs/store/src/remote.rs

```diff
@@ -159,6 +160,73 @@ impl ByteStore {
         Ok(result.map(Bytes::from))
     }
 
+    pub async fn load_bytes_batch(
+        &self,
+        digests: Vec<Digest>,
+        destination: &mut dyn BatchLoadDestination,
+    ) -> Result<HashMap<Digest, Result<bool, String>>, String> {
+        let start = Instant::now();
+        let workunit_desc = format!(
+            "Loading batch at {} of {} digets ({} bytes)",
+            self.instance_name.as_ref().map_or("", |s| s),
+            digests.len(),
+            digests.iter().map(|d| d.size_bytes).sum::<usize>()
+        );
+
+        in_workunit!(
+            "load_bytes_batch",
+            Level::Trace,
+            desc = Some(workunit_desc),
+            |workunit| async move {
+                workunit.increment_counter(Metric::RemoteStoreReadAttempts, 1);
```

Comment:
Currently re-using the metrics from `load`, but unlike `load` this will be the sum of N calls. Should the metric be at the lower per-call level or maybe a different name?